.evg.yml

```diff
@@ -190,8 +169,6 @@ tasks:
     exec_timeout_secs: 3600
     commands:
       - func: "fetch_go114"
-      - func: "fetch_yarn"
-      - func: "fetch_baas_artifacts"
```

Comment:
`test_with_cloud` task will still want the `fetch_baas_artifacts` step included... inside of `setup_baas_server` we have this command:

```cmd
cp baas_artifacts/transpiler baas/etc/transpiler/bin/
```